lib/heroku/cli.rb

```diff
@@ -1,3 +1,8 @@
+if RUBY_VERSION < '1.9.0'
+  $stderr.puts "WARNING: Heroku Toolbelt will require Ruby 1.9+ beginning next week"
+  $stderr.puts "https://github.com/heroku/heroku/pull/1479"
```

Comment:
@dickeyxxx I would have added the actual date (in case someone see this message after the next week). Or, maybe accommodate the message to be more flexible. Something like:

> WARNING: Heroku Toolbelt will start requiring Ruby 1.9+ on MM/DD/YYYY.

Unless, next week you'll update this message to something like:

> WARNING: Heroku Toolbelt requires Ruby 1.9+.